src/coreclr/src/pal/src/libunwind/src/CMakeLists.txt

```diff
@@ -332,11 +343,10 @@ else(CLR_CMAKE_HOST_UNIX)
     endif()
 
     set_source_files_properties(${CLR_DIR}/src/pal/src/exception/remote-unwind.cpp PROPERTIES COMPILE_FLAGS /TP INCLUDE_DIRECTORIES ${CLR_DIR}/src/inc)
-    set_source_files_properties(${CLR_DIR}/src/pal/src/exception/seh-unwind.cpp PROPERTIES COMPILE_FLAGS /TP INCLUDE_DIRECTORIES ${CLR_DIR}/src/inc)
+
     add_library(libunwind_xdac
       OBJECT
       ../../exception/remote-unwind.cpp
-      ../../exception/seh-unwind.cpp
```

Comment:
I assume this is intentional and the cross DAC still compiles.

Reply:
It does.  The reason is I added a copy of the two functions remote-unwind.cpp needed from seh-unwind.cpp to remote-unwind.cpp.